roles/java_internal/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: install java
+  include_role:
+    name: geerlingguy.java
+  vars:
+    java_packages: java-1.8.0-openjdk-devel
+
+
+- name: change java alternative
+  alternatives:
+    name: java
+    path: /usr/lib/jvm/jre-1.8.0-openjdk.x86_64/bin/java
```

Comment:
This is specific to Amazon Linux 1 as Amazon Linux 2 changes the directory name.